test/e2e/common/node/container_probe.go

```diff
@@ -401,7 +401,7 @@ var _ = SIGDescribe("Probing container", func() {
 		cmd := []string{"/bin/sh", "-c", "echo ok >/tmp/health; sleep 10; echo ok >/tmp/startup; sleep 600"}
 		readinessProbe := &v1.Probe{
 			Handler:             execHandler([]string{"/bin/cat", "/tmp/health"}),
-			InitialDelaySeconds: 0,
+			InitialDelaySeconds: 700,
 			PeriodSeconds:       60,
```

Comment:
I meant to put it as a PeriodSeconds. The bug was that probe was waiting for the first PeriodSeconds before running. Now we just call it manually the very first time

Reply:
ah yeah, that's the trick!

Reply:
the fact that we do not respect `InitialDelaySeconds` after the startup probe ended is bothersome...maybe we need to track it as a bug